src/lib/Scenes/Artwork/Components/__tests__/CommercialInformation-tests.tsx

```diff
@@ -62,13 +72,65 @@ describe("CommercialInformation", () => {
         <CommercialInformation artwork={CommercialInformationArtworkNoData} />
       </Theme>
     )
-    expect(component.text()).not.toContain("Sold")
+    expect(component.text()).not.toContain("Contact For Price")
     expect(component.text()).not.toContain("I'm a Gallery")
     expect(component.text()).not.toContain("Consign with Artsy.")
   })
+
+  it("renders seller info correctly for commercial works", () => {
+    const component = mount(
+      <Theme>
+        <CommercialInformation artwork={CommercialInformationArtwork} />
+      </Theme>
+    )
+    expect(
+      component
+        .find(Sans)
+        .at(0)
+        .render()
+        .text()
+    ).toMatchInlineSnapshot(`"Contact For Price"`)
+  })
+  it("renders seller info correctly for non-commercial works", () => {
```

Comment:
This test looks the same as the one above-- was it meant to be updated for the non-commercial case?